main/nginx-httpd.conf

```diff
@@ -4,6 +4,11 @@ lua_shared_dict redirects 10m;
 lua_shared_dict targets 10m;
 # lua_code_cache off;
 
+# Set real ip if request forwarded from VPC (vpc-dc2d9fb9) CIDR
+set_real_ip_from 10.16.0.0/16;
```

Comment:
So all aws VPC availability zones are 10.16? I'm assuming the answer is yes, just confirming.

Reply:
Yes, at least thats what AWS lists for the VPC our load balancers are in.